prisma/migrations/20230126161340_create_table_audit_log_v2/migration.sql

```diff
@@ -0,0 +1,10 @@
+-- CreateTable
+CREATE TABLE "AuditLogV2" (
```

Comment:
w.r.t. merging: We should probably split that pr into several commits. At least 2: 
- 1 for the data migration
- 1 for the changes with the encryption algorithms

Wdyt?

Reply:
Not sure I understand the question, are you referring to this PR? There's no migration happening here whatsoever.

Reply:
Yes, my comment was a little bit confusing. I was referring to the migration of the database in this PR. I would assume that it is a little bit easier to first have one commit that only creates the new table before we actually use it somewhere?

Reply:
Ah, sorry about the confusion. Sure, once we're happy with everything (or at least with the new table) I can push a commit with only the db migration. I wouldn't split it any further though unless you think there's reason to.

Reply:
Nope, I think that would be fine. Thank you